README.md

```diff
@@ -36,9 +36,10 @@ Braces alters the typical WP PHP loop syntax for a more [PHP coding standards] (
 ## Installation
 
 1. If you don't have them already on your system, install all required gems above
-2. `cd` into `sass` directory run `bourbon install` run `neat install`
-3. `cd` into `root` directory run `npm install`
-4. run `gulp`
+2. `cd` into `sass/libraries/bourbon` directory run `git submodule init` then run `git submodule update`
+3. `cd` into `sass/libraries/neat` directory run `git submodule init` then run `git submodule update`
```

Comment:
These instructions can be consolidated into one command run from the root directory: `git submodule update --recursive --init`